mappings/net/minecraft/advancement/Advancement.mapping

```diff
@@ -16,6 +25,7 @@ CLASS net/minecraft/unmapped/C_kdwyuhdb net/minecraft/advancement/Advancement
 		ARG 1 style
 	METHOD m_egdqigyx createTask ()Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
 	METHOD m_ehjzgofc getId ()Lnet/minecraft/unmapped/C_ncpywfca;
+	METHOD m_ggypmlkv doesSendTelemetryEvent ()Z
```

Comment:
oh, about this rename, Advancement$Task should match these new names

Reply:
I do not see any discrepancies between Task and Advancement?

Reply:
I've mapped the `doesSendTelemetryEvent` stuff as `sendsTelemetryEvent` there, like you've done originally, but now that it has been renamed, the names don't match anymore

Reply:
You mapped the field and parameters, what's name with the `does` prefix is the getter not the field.

Reply:
oh dear, i thought that there was an equivalent method